ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -552,6 +555,10 @@ ghciDef :: GhcFlavor -> String
 ghciDef GhcMaster = ""
 ghciDef _ = "-DGHCI"
 
+ghcStageDef :: GhcFlavor -> String
+ghcStageDef GhcMaster = "-DGHC_STAGE=2"
```

Comment:
This is definitely not the right thing to do now that this is defined in the header. We shouldn’t pass any define flags now, see #149 for some alternatives.